test/RibbonThetaYearnVault.ts

```diff
@@ -4027,6 +3770,29 @@ function behavesLikeRibbonOptionsVault(params: {
         assert.bnEqual(await position.shares, BigNumber.from(0));
       });
     });
+
+    describe("#yearnPause", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("yearnPaused initially set to true", async function () {
```

Comment:
`it` statements should read like a sentence.

We would change this to:
`it("sets yearnPaused to true initially`